Sources/TuistKit/Services/WhoamiService.swift

```diff
@@ -43,6 +39,10 @@ final class SessionService: SessionServicing {
         }
         let config = try await configLoader.loadConfig(path: directoryPath)
         let serverURL = try serverURLService.url(configServerURL: config.url)
-        try await serverSessionController.printSession(serverURL: serverURL)
+        if let whoami = try await serverSessionController.whoami(serverURL: serverURL) {
+            logger.notice("\(whoami)")
+        } else {
+            logger.notice("You are not logged in. Run tuist auth to log in.")
```

Comment:
Technically, we now show this also when you are authenticated with older tokens that didn't have the email claim. Given user can just run `tuist auth` in that case to get the new token, I think that's fine and simplifies our code quite a bit since we don't have to deal with the old tokens.